Octokit.Tests.Integration/Reactive/ObservableFollowersClientTests.cs

```diff
@@ -0,0 +1,305 @@
+﻿using System.Reactive.Linq;
+using System.Threading.Tasks;
+using Octokit.Reactive;
+using Xunit;
+
+namespace Octokit.Tests.Integration.Reactive
+{
+    public class ObservableFollowersClientTests
+    {
+        public class TheGetAllForCurrentMethod
+        {
+            readonly ObservableFollowersClient _followersClient;            
+
+            public TheGetAllForCurrentMethod()
+            {
+                var github = Helper.GetAuthenticatedClient();
+
+                _followersClient = new ObservableFollowersClient(github);
+            }
+
+            [IntegrationTest]       
+            public async Task ReturnsFollowers()
+            {
+                var followers = await _followersClient.GetAllForCurrent().ToList();
```

Comment:
Oh yes, this isn't your fault. I'm just thinking aloud here about how to make these tests more adaptable to other contributors...